### ISSUE TITLE
Question: typo error in g2/error.h?

### ISSUE THREAD

**Issue by gilbertoverg** [synthetos/g2#275]
Hi, in g2/error.h at line 338:
```
#define STAT_SOFT_LIMIT_EXCEEDED_BMAX 220       // soft limit error - B maximum
```
is the value 220 correct?

**Comment by aldenhart** [comment]
It's an error. It should be 230. This will be included in a subsequent PR.

**Comment by justinclift** [comment]
Closing this, as it was fixed recently with 4608936af1f0ba5525378f1f8c5f189136393bdc. :smile:
